src/logistration/RegistrationFailure.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+
+import Alert from './Alert';
```

Comment:
Can we start using paragon alert for any new implementation?

Reply:
This will be done in another ticket.

Reply:
I believe that will increase the technical debt and we should avoid it for any new changes.

Reply:
This was decided for any future work. Not for any existing work.

Reply:
Done